src/clj/rems/db/blacklist.clj

```diff
@@ -40,6 +42,30 @@
 (defn add-event! [event]
   (db/add-blacklist-event! {:eventdata (event->json event)}))
 
+(defn- command->event [command user-id]
+  {:event/actor user-id
+   :event/time (time/now)
+   :userid (get-in command [:blacklist/user :userid])
+   :resource/ext-id (get-in command [:blacklist/resource :resource/ext-id])
+   :event/comment (:comment command)})
+
+(defn- validate-event [event]
```

Comment:
we have `validate-event` and on the other hand we have `(s/validate BlacklistEvent event)`... is there a risk of confusion between these two types of validation? should one be called something else?